src/roles/references.ts

```diff
@@ -9,12 +9,17 @@ export class Eq extends Role {
     const open = new this.state.Token("ref_open", "a", 1)
     const content = new this.state.Token("text", "", 0)
     const close = new this.state.Token("ref_close", "a", -1)
-    resolveRefLater(this.state, { open, content, close }, data.content, {
-      kind: TargetKind.equation,
-      contentFromTarget: target => {
-        return `(${target.number})`
+    resolveRefLater(
+      this.state,
+      { open, content, close },
+      { kind: "eq", label: data.content },
```

Comment:
I have renamed the state to "label" rather than name, which is more consistent with mdast.

No external changes for this rename.